ml.net/InclusiveCodeReviews.ConsoleApp/Program.cs

```diff
@@ -1,5 +1,8 @@
 using System;
```

Comment:
So this `InclusiveCodeReviews.ConsoleApp` was useful on its own for testing the model.

Could we make a new project like `InclusiveCodeReviews.Scoring` that is a completely new console app? It can be in the same solution that would be fine. Thanks!

Reply:
Sure, added.